src/providers/token-properties-provider.ts

```diff
@@ -56,8 +56,7 @@ export class TokenPropertiesProvider implements ITokenPropertiesProvider {
     const tokenToResult: TokenPropertiesMap = {};
 
     if (
-      !providerConfig?.enableFeeOnTransferFeeFetching ||
-      this.chainId !== ChainId.MAINNET
```

Comment:
this is now using GQL (with onChain fallback) - i believe GQL now supports all chains, correct?

Reply:
exactly. it's in case GQL fallback